lib/ansible/modules/debconf.py

```diff
@@ -209,15 +207,17 @@ def main():
         if vtype is None or value is None:
             module.fail_json(msg="when supplying a question you must supply a valid vtype and value")
 
+        # ensure we compare booleans supplied to the way debconf sees them (true/false strings)
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
Will do.  When modifying a PR is the standard practice in this repo to add another commit that will be squashed, or to rewrite the original commit to be complete?

Reply:
adding another commit is perfectly fine.